pkg/registry/client.go

```diff
@@ -134,22 +134,28 @@ interpret:
 			Created time.Time `json:"created"`
 			OS      string    `json:"os"`
 			Arch    string    `json:"architecture"`
+		}
+		if err = json.Unmarshal([]byte(man.History[0].V1Compatibility), &v1); err != nil {
+			return ImageEntry{}, err
+		}
+
+		var config struct {
```

Comment:
Please add a comment on why the config is unmarshaled separately. Also, it would be a good idea to add comment to each structure definition, linking to their authoritative source (e.g. OCI image standard )

Reply:
The authoritative sources of the image labels are already recorded in `image/image.go`. I have in-lined comments for the others.